<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>210</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>210</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Sustituci√≥n de clases en etiquetas</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="Formulario">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>481</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string>Cambio de clase en labels</string>
    </property>
    <widget class="QPushButton" name="SelectLabelsDir">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>161</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Directorio de etiquetas</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="labelsDir">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>441</width>
       <height>21</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Seleccione directorio de etiquetas</string>
     </property>
    </widget>
    <widget class="QLabel" name="CambiarLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>85</y>
       <width>58</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Cambiar</string>
     </property>
    </widget>
    <widget class="QLabel" name="porLabel">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>86</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Por</string>
     </property>
    </widget>
    <widget class="QPushButton" name="cambiarButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>441</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Cambiar clases</string>
     </property>
    </widget>
    <widget class="QComboBox" name="OrigenComboBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>80</y>
       <width>161</width>
       <height>32</height>
      </rect>
     </property>
     <property name="editable">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QComboBox" name="DestinoComboBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>80</y>
       <width>161</width>
       <height>32</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
